### DISCUSSION TITLE
Sample Server and Client

### DISCUSSION THREAD

**Discussion by myokeeh** [CommunityToolkit/Datasync#175] [Q&A] [answered]
At this point, are the sample projects/solution in https://github.com/CommunityToolkit/Datasync/tree/main/samples a good starting point? 

I've looked at a few references and just want to avoid going in circles or doing something "stale." Specifically, is the authentication implementation in the sample "better" than what's in https://github.com/CommunityToolkit/Datasync/blob/main/docs/content/in-depth/client/auth.md or https://communitytoolkit.github.io/Datasync/in-depth/client/auth/index.html?

https://apps-on-azure.net/datasync/2024/2024-10-08-datasync-part1.html
https://communitytoolkit.github.io/Datasync/

**Comment by adrianhall** [comment, marked as answer]
I've been working on a more complex sample that has multiple tables, joins, authentication, etc. with a working web, WinUI3, and Android implementation.  It's definitely not there yet (which is why I haven't also written the client-side blog posts).

Samples will always somewhat go stale - I try to keep them up to date as best I can.
